Cargo.toml

```diff
@@ -69,8 +70,12 @@ web-time = { version = "1.1.0" }
 wasm-bindgen-futures = "0.4.18"
 
 [features]
-default = ["fs"]
-cloud = ["serde", "serde_json", "quick-xml", "hyper", "reqwest", "reqwest/stream", "chrono/serde", "base64", "rand", "ring", "http-body-util", "form_urlencoded", "serde_urlencoded"]
+default = ["fs", "crypto-ring"]
+# Crypto provider selection - choose one
+crypto-ring = ["ring"]
+crypto-aws-lc-rs = ["aws-lc-rs"]
+# Cloud features (requires a crypto provider to be selected separately)
+cloud = ["serde", "serde_json", "quick-xml", "hyper", "reqwest", "reqwest/stream", "chrono/serde", "base64", "rand", "http-body-util", "form_urlencoded", "serde_urlencoded"]
```

Comment:
This is a breaking change if someone has `no_default_features=true, features=["aws"]`. Not very nice, but if mutual exclusion is wanted I don't see a way around this.